plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/delete/DeleteFile.java

```diff
@@ -38,16 +41,27 @@
     private final long recordCount;
     private final long fileSizeInBytes;
     private final List<Integer> equalityFieldIds;
+    private final Map<Integer, byte[]> lowerBounds;
+    private final Map<Integer, byte[]> upperBounds;
 
     public static DeleteFile fromIceberg(org.apache.iceberg.DeleteFile deleteFile)
     {
+        Map<Integer, byte[]> lowerBounds = deleteFile.lowerBounds() == null ?
+                ImmutableMap.of() :
+                deleteFile.lowerBounds().entrySet().stream().collect(toImmutableMap(Map.Entry::getKey, entry -> entry.getValue().array()));
+        Map<Integer, byte[]> upperBounds = deleteFile.upperBounds() == null ?
+                ImmutableMap.of() :
+                deleteFile.upperBounds().entrySet().stream().collect(toImmutableMap(Map.Entry::getKey, entry -> entry.getValue().array()));
```

Comment:
> `.array()`

do we need to make a defensive copy of these?

Reply:
Probably. Added a call to `clone`